File: AoC2019Tests/Day02Test.fs
module Day02Test

open AoC2019
open NUnit.Framework

[<TestCase("1,0,0,0,99","2,0,0,0,99")>] 
[<TestCase("2,3,0,3,99","2,3,0,6,99")>]
let Part1 (input: string, expected : string) =
    Day02.input <- fun _ -> input.Split(',') :> seq<string>
    let actual = Day02.SolvePart1
    Assert.AreEqual(expected, actual)

[<Test>]
let Part2 () = 
    let actual = Day02.SolvePart2
    Assert.AreEqual("", actual)

File: AoC2019Tests/Day02Test.cs
namespace AoC2019Tests;

[Ignore("Included in IntCodeComputerTests")]
public class Day02Test
{
    Day02 day02;
            
    [SetUp]
    public void Setup( )
    {
        day02 = new Day02("day02test1");
    }
    
    [Test]
    public async Task Part1( )
    {
        var actual = await day02.SolvePart1( );
        Assert.That(actual, Is.EqualTo("1"));
    }
}
